From cc33459c3afbfb2fc12aeede749eb37c286096fe Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Fri, 19 Aug 2022 16:37:31 +0000
Subject: [PATCH] Disable Generated Kernel Headers

---
 build/soong/Android.bp | 14 +-------------
 1 file changed, 1 insertion(+), 13 deletions(-)

diff --git a/build/soong/Android.bp b/build/soong/Android.bp
index 64a82dd0..0df8f7d6 100644
--- a/build/soong/Android.bp
+++ b/build/soong/Android.bp
@@ -20,9 +20,6 @@ bootstrap_go_package {
 awaken_generator {
     name: "generated_kernel_includes",
 
-    // The headers make command
-    cmd: "$(PATH_OVERRIDE_SOONG) $(KERNEL_MAKE_CMD) $(KERNEL_MAKE_FLAGS) -C $(TARGET_KERNEL_SOURCE) O=$(KERNEL_BUILD_OUT_PREFIX)$(genDir) ARCH=$(KERNEL_ARCH) $(KERNEL_CROSS_COMPILE) headers_install",
-
     // Directories that can be imported by a cc_* module generated_headers property
     export_include_dirs: [
         "usr/audio/include/uapi",
@@ -30,16 +27,7 @@ awaken_generator {
         "usr/include/audio",
         "usr/include/audio/include/uapi",
         "usr/techpack/audio/include",
-    ],
-
-    // Sources for dependency tracking
-    dep_root: "$(TARGET_KERNEL_SOURCE)",
-    dep_files: [
-        "Makefile",
-        "include/**/*",
-        "arch/$(KERNEL_ARCH)/include/**/*",
-        "techpack/audio/include/**/*",
-    ],
+    ]t,
 }
 
 cc_library_headers {
-- 
2.17.1

